packages/base/src/features/OpenUI5Support.ts

```diff
@@ -103,6 +104,7 @@ class OpenUI5Support {
 		const config = core.getConfiguration();
 		core.attachThemeChanged(async () => {
 			await setTheme(config.getTheme());
+			await setThemeRoot(config.getThemeRoot());
```

Comment:
I think that we don't need that here. In mixed scenario where OpenUI5 / SAPUI5 is used, the external theme will be loaded by OpenUI5 / SAPUI5 core and we have only to detect the external theme when our callback for themeChanged is called.